### PR TITLE
Add permission provider table creation missing quries

### PR DESCRIPTION
**Pull request by ksdperera**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/internal/dao/PermissionsDAO.java [hunk 1]
```diff
@@ -53,6 +53,80 @@ public PermissionsDAO(DataSourceService dataSourceService, PermissionConfig perm
...
+        ResultSet rs = null;
+        try {
+            conn = getConnection();
+            query = queryManager.getQuery(conn, QueryManager.TABLE_CHECK_QUERY);
+            stmt = conn.prepareStatement(query.replace(QueryManager.TABLE_NAME_PLACEHOLDER, tableName));
+            rs = stmt.executeQuery();
+            return true;
+        } catch (SQLException e) {
+            if (log.isDebugEnabled()) {
+                log.debug("Table '" + tableName + "' assumed to not exist since its existence check resulted "
```
Comment (this) [inline components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/internal/dao/PermissionsDAO.java]: Any reason why we cannot use parameterized log messages (as in [above](https://github.com/wso2/carbon-analytics-common/pull/539/files#diff-fc89ca3f25c2f3625c314b55f42f57e0R97))?

Comment (ksdperera) [reply components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/internal/dao/PermissionsDAO.java]: Yes, we can parameterize, there was no special reason.
Please find the fixes in commit: https://github.com/wso2/carbon-analytics-common/pull/539/commits/44db61379083d6c990cc1ef4241eb9e8129f7ecb

File: components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/internal/dao/PermissionsDAO.java [hunk 2]
```diff
@@ -53,6 +53,80 @@ public PermissionsDAO(DataSourceService dataSourceService, PermissionConfig perm
...
+        String query = null;
+        ResultSet rs = null;
+        try {
+            conn = getConnection();
+            query = queryManager.getQuery(conn, QueryManager.TABLE_CHECK_QUERY);
+            stmt = conn.prepareStatement(query.replace(QueryManager.TABLE_NAME_PLACEHOLDER, tableName));
+            rs = stmt.executeQuery();
+            return true;
+        } catch (SQLException e) {
+            if (log.isDebugEnabled()) {
```
Comment (this) [inline components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/internal/dao/PermissionsDAO.java]: With paramterized log messages, we don't need `log.isDebugEnabled()` as SLF4J internally checks it before composing the log essage

Comment (ksdperera) [reply components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/internal/dao/PermissionsDAO.java]: Please find the fixes: f316cd954ac9474be6cff0fd5e8caecd2a72404b

**Pull request merged**